src/OptionsRESTController.php

```diff
@@ -338,6 +352,20 @@ public function get_item_schema() {
 					],
 				],
 			];
+
+			/**
+			 * Filters schema for services to add additional items.
+			 *
+			 * @internal
+			 *
+			 * @param array $schema Schema.
+			 */
+			$services_schema = apply_filters( 'amp_rest_options_schema', [] );
```

Comment:
Filters should always use extra hardening:
```suggestion
			$services_schema = array_filter( (array) apply_filters( 'amp_rest_options_schema', [] ) );
```